### PR TITLE
Added a base websocket exception.

### PR DESCRIPTION
**Pull request by JakeMakesStuff**
This could be useful in situations where you want to catch all websocket exceptions.

### CODE REVIEW

**Comment by JakeMakesStuff** [comment]
Hmm, not quite sure why Circle CI failed.

File: src/websockets/exceptions.py [hunk 1]
```diff
@@ -20,7 +21,14 @@
 ]
 
 
-class InvalidHandshake(Exception):
+class WebsocketError(Exception):
```
Comment (messense) [inline src/websockets/exceptions.py]: Should it be `WebSocketError`?

Comment (aaugustin) [reply src/websockets/exceptions.py]: For consistency with other APIs, yes.

**Comment by JakeMakesStuff** [comment]
Sorry for the merge conflict, I just pushed a fix for that.

**Comment by aaugustin** [comment]
Thanks for your contribution!

I wanted to take this opportunity to rationalize the exception tree, which I just did in 8f1db07bb81e60c10e1d0a3efedfbbeb08962ac5.

You'll have `WebSocketException` in version 8.0, which I hope to release within a few days.

**Comment by JakeMakesStuff** [comment]
Thanks @aaugustin, it's appreciated for the workflow which I have. :)

**Pull request closed without merging**